lib/util.js

```diff
@@ -6,11 +6,14 @@ var appPort = parseInt(argv.mocking_port, 10);
 // +1 is used for selennium
 var mocksPort = appPort + 2;
 
+var mocksHttpsPort = appPort + 3;
```

Comment:
The port spacing is flexible with the parameter - base_port_spacing (https://github.com/TestArmada/magellan/blob/bde5dbf40486aac08835101af7fb6a35b1eef68e/src/settings.js#L60) 

We need both http as well as https.